Code/GraphMol/ChemReactions/ReactionRunner.cpp

```diff
@@ -48,11 +48,25 @@ namespace RDKit {
 typedef std::vector<MatchVectType> VectMatchVectType;
 typedef std::vector<VectMatchVectType> VectVectMatchVectType;
 
+namespace {
```

Comment:
There really is none, except we have the common_properties namespace to

1. protect typos ( I just realized that one of the BalabanJ entries is due to a typo from the original code that was automatically converted, there should only be one which kind of proves this point :) )
2. Eliminate the cost of converting a const char * into a std::string which was happening before the namespace was added.

These two const strings are not common, they are just used privately so they weren't added to the common_properties namespace, but they serve the same purpose as "typos" are caught by the compiler.